README.md

```diff
@@ -119,8 +124,6 @@ MIT Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-a
 | tags | A map of tags to add to all resources. | map | `<map>` | no |
 | vpc_id | VPC where the cluster and workers will be deployed. | string | - | yes |
 | worker_additional_security_group_ids | A list of additional security group ids to attach to worker instances | list | `<list>` | no |
-| worker_group_count | The number of maps contained within the worker_groups list. | string | `1` | no |
-| worker_groups | A list of maps defining worker group configurations. See workers_group_defaults for valid keys. | list | `<list>` | no |
```

Comment:
To make the ever growing list of variables easier to read, perhaps let's keep them grouped like this?

```
worker_groups
worker_group_count
worker_groups_launch_template
worker_group_launch_template_count
```

Reply:
Fixed